certmanager/README.md

```diff
@@ -11,3 +11,15 @@ in a standalone manner, separate from an SSL/TLS connection.
 $ make
 $ ./certverify
 ```
+## Verification of OQS Falcon Certificates
+
+The `generate_falcon_chains.sh` script will allow you to use the OQS project's
```

Comment:
Would it be useful to provide a link to the libOQS openssl project?

Reply:
Will add reference to wolfssl/INSTALL.